skcosmo/decomposition/_kernel_pcovr.py

```diff
@@ -279,18 +296,16 @@ def fit(self, X, Y, Yhat=None, W=None):
             to have unit variance, otherwise :math:`\\mathbf{Y}` should be
             scaled so that each feature has a variance of 1 / n_features.
 
-        Yhat: ndarray, shape (n_samples, n_properties), optional
-            Regressed training data, where n_samples is the number of samples and
-            n_properties is the number of properties. If not supplied, computed
-            by ridge regression.
-
         Returns
         -------
         self: object
             Returns the instance itself.
 
         """
 
+        if not any([self.regressor is None, isinstance(self.regressor, KernelRidge)]):
```

Comment:
Why this syntax? For simple two-component logic, it would be more typical to write
```suggestion
        if self.regressor is not None and not isinstance(self.regressor, KernelRidge):
```

I find it much easier to read and understand

Reply:
I just used this syntax because it's consistent with what I had in PCovR, where there are more valid regressor possibilities. I have no problem changing it.